content/en/cloud_cost_management/_index.md

```diff
@@ -273,21 +273,30 @@ You can visualize your ingested data using the following cost types:
+
+There are three types of rules supported: **Add tag**, **Alias tag keys**, and **Map multiple tags**. You can keep your rules organized by leveraging rules-sets, which act as folders for your rules. The rules are executed in order (from top to bottom), to keep the execution order deterministic. You can organize rules and rulesets to ensure the order of execution matches your business logic.
+
+**Add tag** - Add a new tag (key + value) based on the presence of existing tags on Cloud Costs data. 
+
+For example, you can create a rule to tag all resources with their business unit based on the services those resources are a part of. 
+{{< img src="cloud_cost/tags_addnew.png" alt="Add new business unit tag to resources with service:processing, service:creditcard, or service:payment-notification." >}}
+
+**Alias tag keys** - Map existing tag values to a more standardized tag. 
 
-There are two types of rules supported: **Create new tag**, and **Alias existing tag keys**. You can keep your rules organized by leveraging rules-sets, which act as folders for your rules. The rules are executed in order (from top to bottom), to keep the execution order deterministic. You can organize rules and rulesets to ensure the order of execution matches your business logic.
+For example, if your organization wants to use the standard `application` tag key, but several teams have a variation of that tag (like `app`, `webapp`, or `apps`), you can alias `apps` to `application`. Each alias tag rule allows you to alias a maximum of 25 tag keys to a new tag.
 
-**Create new tag** - This allows you to create a new tag (key + value) based on the presence of existing tags. For example, you can create a rule to tag all resources that are part of team A, B, or C, and also run a specified application, with a new `cost-center:webstore` tag.
+The rule stops executing for each resource after a first match is found. For example, if a resource already has a `app` tag, then the rule no longer attempts to identify a `webapp` or `apps` tag.
+{{< img src="cloud_cost/tags_alias.png" alt="Add application tag to resources with app, webapp, or apps tag." >}}
 
-**Alias existing tag keys** - This allows you to use values from an existing tag, to map to a more standardized tag key. For example, if you're looking to standardize across your organization to use the `application` tag key, but several teams have a variation of that tag (like `app`, `web-app`, or `apps`), you can alias `apps` to `application`. Each alias tag rule allows you to alias a maximum of 25 tag keys to a new tag.
+**Map multiple tags** - Use [Reference Tables][2] to add multiple tags to cost data without needing to create multiple rules. This will map the values from your Reference Table's primary key column to values from cost tags. If found, it will add the selected Reference Table columns as tags to cost data.
```

Comment:
By "it" do you mean the pipeline or the Datadog application?
```suggestion
#### Map multiple tags

Use [Reference Tables][2] to add multiple tags to cost data without creating multiple rules. This will map the values from your Reference Table's primary key column to values from cost tags. If found, the pipelines adds the selected Reference Table columns as tags to cost data.
```